pci.ids

```diff
@@ -57,7 +57,7 @@
 
 # Accelerators
 #------------
-
+1002  Advanced Micro Devices, Inc. [AMD/ATI]
```

Comment:
Maybe add some devices here? I don't like empty manufacturers in the list of validated equipment.

Reply:
Removed this and fixed the rest.